packages/codemod/src/upgrade-legacy.ts

```diff
@@ -112,6 +112,29 @@ const upgradeLegacy = async () => {
       fs.writeFileSync(path.resolve("next.config.js"), program.toSource())
     },
   })
+  
+  steps.push({
+    name: "update .eslintrc.js configuration",
+    action: async () => {
+      if (fs.existsSync(path.resolve(".eslintrc.js"))) {
+        const program = getCollectionFromSource(".eslintrc.js")
+        const parsedProgram = program.get()
+        parsedProgram.value.program.body = []
+        const moduleExport = j.expressionStatement(
+          j.assignmentExpression(
+            "=",
+            j.memberExpression(j.identifier("module"), j.identifier("exports")),
+            j.callExpression(j.identifier("require"), [j.identifier(`"@blitzjs/next/eslint"`)]),
+          ),
+        )
+        parsedProgram.value.program.body.push(moduleExport)
+        fs.writeFileSync(path.resolve(".eslintrc.js"), program.toSource())
+      }
+      else{
+        log.error(".eslintrc.js does not exist")
+      }
```

Comment:
I don't think we need to show an error if .eslint doesn't exist.

But if you think we should, instead of logging the error here, can we show it at the end?

You can use the `collectedErrors` array, and inside the else statement just do:

```ts
          collectedErrors.push({
            message:
              ".eslintrc.js does not exist",
            step: stepIndex,
          })
```

`stepIndex` is a parameter inside the action function.

Reply:
> I don't think we need to show an error if .eslint doesn't exist.
> 
@dillonraphael I had added that check because when I tested it with a legacy repository without the .eslint config 

![image](https://user-images.githubusercontent.com/83594610/185538095-4901d637-81f2-4508-b8d0-c0fe971e14e9.png)
The above error will stop the codemod even though the config is not technically essential. 

![image](https://user-images.githubusercontent.com/83594610/185538939-95e53e51-eb45-4701-a9ce-fa5369a77339.png)
So I had used the format done when cookie prefix is not defined in the blitz config file

**This is how it would look to a user when they did not have .eslintrc.js file**
![image](https://user-images.githubusercontent.com/83594610/185538861-8320763b-6ed7-49fe-9eef-62c89a101130.png)

Reply:
> But if you think we should, instead of logging the error here, can we show it at the end?
> 
> You can use the `collectedErrors` array, and inside the else statement just do:
> 
> ```ts
>           collectedErrors.push({
>             message:
>               ".eslintrc.js does not exist",
>             step: stepIndex,
>           })
> ```
> 
> `stepIndex` is a parameter inside the action function.

Okay I shall add a commit with the requested change.